usaspending_api/recipient/models.py

```diff
@@ -62,12 +62,12 @@ class HistoricParentDUNS(models.Model):
     Model representing DUNS data (imported from the broker)
     """
 
-    awardee_or_recipient_uniqu = models.TextField(primary_key=True)
-    legal_business_name = models.TextField()
-    ultimate_parent_unique_ide = models.TextField()
-    ultimate_parent_legal_enti = models.TextField()
+    awardee_or_recipient_uniqu = models.TextField(primary_key=True, db_index=True)
```

Comment:
Neither this index nor the index on `year` has been used. Might be better to drop from the DB